SS_selex.tpl

```diff
@@ -1,7 +1,7 @@
 // SS_Label_file  #11. **SS_selex.tpl**
 // SS_Label_file  # * <u>get_selectivity()</u>  // does length and age selectivity and retention for all fleets
 // SS_Label_file  # * <u>make_fish_selex()</u>  // for all fleets in a particular season, does the dot product of length distribution with length selectivity and retention vectors to calculate equivalent mean quantities at age for each platoon
-
+

```

Comment:
something weird is going on with the line endings. I will try to fix.

Reply:
Github told me when I tried to edit inline that this file had a mix of line endings, which is why this is happening. It is changing them all to windows style (CRLF, which I think our other files are also using). So I think it is ok as is, as even editing this file on github would result in these changes.